vulnerabilities/data_dump.py

```diff
@@ -68,3 +71,64 @@ def ubuntu_dump(html):
             vulnerability=vulnerability,
             package=package
         )
+
+
+def archlinux_dump(extract_data):
+    """
+    Save data scraped from archlinux' security tracker.
+    """
+    for item in extract_data:
+        cves = item['issues']
+        advisories = item['advisories']
+        group = item['name']
+
+        unique_advisories = []
```

Comment:
Please use set

Reply:
@singh1114 Since, set is an unordered collection, I was getting errors while writing unit tests so I switched to this way.